project_event/models/project_task.py

```diff
@@ -427,34 +427,55 @@ def update_reservation_event(self, vals):
         if self.reservation_event_id:
             reservation_event = self.env['calendar.event'].browse(
                 self.reservation_event_id)
-            update_vals = {}
```

Comment:
Add the unit tests for this

Reply:
@rimbendhaou : Created a new task for writing test cases for this task. You can merge this code.

Reply:
@rimbendhaou : check in the pending pull in the list (related with Refactoring update_reseration_event)